README.md

```diff
@@ -22,12 +20,11 @@ How?
 server.register([
   register: require('ot-hapi-queryparams'),
   options: {
-  delete_original: true/false // remove the origin query param attribute
+  deleteOriginal: true/false // remove the origin query param attribute
   //ONLY ONE OPTION SHOULD BE SET TO TRUE
-    camel_case: true, // user_name => userName , UserName => userName
-    all_caps: true,   // username => USERNAME
-    lower_case: true, // UserName => username
-    first_cap: true, // userName => UserName
+    camelCase: true, // username => userName (given a schema with userName as a key)
+    lowerCase: true, // UserName => username
+  joiSchema: {} // required for camelCase
```

Comment:
instead of making joiSchema separate, why don't you passing the schema as an option to `camelCase`? 

so instead of `camelCase: true`, you'd pass `camelCase: {joiSchema}`